admin/test/indexes/TagPagesTest.scala

```diff
@@ -39,34 +49,42 @@ import test.WithTestExecutionContext
     )
 
     for ((unicode, ascii) <- words) {
-      tagPages.alphaIndexKey(unicode) shouldEqual ascii
+      alphaIndexKey(unicode) shouldEqual ascii
     }
   }
 
-  it should "index by 0-9 if the first character is a digit" in {
+  it should "index as 'numeric' if the first character is a digit" in {
     val fixtures = Seq(
       "100 Years of Solitude",
       "1984",
       "20,000 Leagues Under the Sea",
+      "0 hero: a very special number",
     )
 
     for (fixture <- fixtures) {
-      tagPages.alphaIndexKey(fixture) shouldEqual "1-9"
+      alphaIndexKey(fixture) shouldEqual "1-9"
     }
   }
 
-  "byWebTitle" should "convert an enumerator of tags into a Future of alpha-ordered TagPages" in {
-    def tagFixture(webTitle: String): ApiTag =
-      ApiTag(
-        "id/id",
-        TagType.Type,
-        None,
-        None,
-        webTitle,
-        "",
-        "",
-      )
```

Comment:
This `tagFixture()` method has been moved out of the individual scope of this test case, and up to the level of the class, so that it can be used in other test cases (like _"not crash if a tag has no last name or first name"_)